source/release-notes/backend.txt

```diff
@@ -13,6 +13,23 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240724:
+
+24 July 2024 Release
+~~~~~~~~~~~~~~~~~~~~
+
+- GraphQL and Static Hosting Deprecation 
+
+  - Apps that use GraphQL or Static Hosting will now display banners in their App Services 
+    UI alerting customers of both the GraphQL and Static Hosting deprecations.
```

Comment:
do we want to link out to the [related docs page](https://www.mongodb.com/docs/atlas/app-services/migrate-hosting-graphql/#std-label-migrate-hosting-graphql)? Something like:
```suggestion
  - Apps that use GraphQL or Static Hosting will now display banners in their App Services 
    UI alerting customers of both the GraphQL and Static Hosting deprecations. For more information, refer to the [Migrate Static Hosting and GraphQL From App Services] page.
```